cmd/litestream/main.go

```diff
@@ -323,6 +324,10 @@ type ReplicaConfig struct {
 	User     string `yaml:"user"`
 	Password string `yaml:"password"`
 	KeyPath  string `yaml:"key-path"`
+
+	// Encryption identities and recipients
+	AgeIdentities []string `yaml:"age-identities"`
+	AgeRecipients []string `yaml:"age-recipients"`
```

Comment:
It'd be nice to reduce the stutter and make the config like:

```yml
age:
  identities: ...
  recipients: ...
```

It can just be an anonymous struct in the config:

```go
type ReplicaConfig struct {
	...
	
	Age struct {
		Identities []string `yaml:"identities"`
		Recipients []string `yaml:"recipients"`
	} `yaml:"age"`
}
```